service/src/main/java/bio/terra/pipelines/app/configuration/internal/WdlPipelineConfiguration.java

```diff
@@ -0,0 +1,12 @@
+package bio.terra.pipelines.app.configuration.internal;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "pipelines.wdl")
+@Getter
+@Setter
+public class WdlPipelineConfiguration {
```

Comment:
noticing you named this pretty generically. what else do you see adding to this configuration besides `quotaConsumedPollingIntervalSeconds`?

Reply:
not sure but i didnt want it associated with imputation - happy to put it in an already existing configuration that wasnt imputations.  I imagine there will be a couple of things we'll want to move out of imputation config to this one once we get our next wdl pipeline.

Reply:
yeah sounds good. no need to combine with an existing configuration.